### PR TITLE
add new enum value to specimen_type

### PR DESCRIPTION
**Pull request by Buwujiu**

### CODE REVIEW

**Review by joneubank — APPROVED** [review]
Changes look fine, look like they follow the established pattern (which we should review later).

Note that there are issues running Song with security enabled since the Spring upgrade, so don't merge this to master yet. I need to follow up on those errors. This will fail to start in the overture QA env but not because of your changes.

**Review by blabadi — APPROVED** [review]

**Comment by Buwujiu** [comment]
Thanks for the headsup @joneubank !

**Pull request merged**
